python-sdk/tests_integration/databases/test_all_databases.py

```diff
@@ -276,9 +310,10 @@ def test_load_pandas_dataframe_to_table_with_replace(database_table_fixture):
         {"database": Database.DELTA, "table": Table()},
         {"database": Database.MSSQL, "table": Table(metadata=Metadata(schema=SCHEMA))},
         {"database": Database.DUCKDB, "table": Table()},
+        {"database": Database.MYSQL, "table": Table()},
     ],
     indirect=True,
-    ids=["bigquery", "postgres", "snowflake", "sqlite", "delta", "mssql", "duckdb"],
+    ids=["bigquery", "postgres", "snowflake", "sqlite", "delta", "mssql", "duckdb", "mysql"],
```

Comment:
I feel like we could set some Global setting of "all databases".  It just feels like we use the same lines over and over again in these tests

Reply:
these are not "all databases", redshift is missing in above :-)

Reply:
not sure about that , @pankajkoti might know the reason as he introduced redshift support